<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifClient</class>
 <widget class="QDialog" name="ModifClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>495</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nom *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineNom"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Prenom *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePrenom"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Adresse *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineAdresse"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ville *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineVille"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>CP *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinCP">
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Jour de rendez-vous *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateRDV"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Duree consultation (minutes) *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinDuree">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Priorité *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinPriorite">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Affectation *</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboAffectation">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Téléphone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineTelephone"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Commentaires</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textCommentaire"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>* Champs obligatoires</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="bouttonOK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bouttonAnnuler">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
